### DISCUSSION TITLE
rutorrent 4.2.7 File Manager create archive error

### DISCUSSION THREAD

**Discussion by InsideTheMindofThePuzzledOne** [Novik/ruTorrent#2595] [Q&A]
Hello everyone, 1st post so hope this is the right place. Using rutorrent 4.2.7 and trying to use file manager to create .rar archive of our families camping trip videos. But when i right click them under file manager and select create new archive and set the type to RAR and the compression to store and the volume size (in mb) to 300mb and click create i get this error message.

Argument 1 passed to Flm\Archive::getCompressBin() must be of the type string, null given, called in /srv/rutorrent/plugins/filemanager/src/Archive.php on line 32

Not sure if i need a plugin or an update or what exactly the problem is.

Thanks for any aid offered.

**Comment by TrimmingFool** [comment]
There is already an issue for this: https://github.com/nelu/rutorrent-filemanager/issues/36

It looks like nelu and Micdu70 were maintaining this plugin: https://github.com/nelu/rutorrent-filemanager/graphs/contributors
